assets/js/preview.js

```diff
@@ -0,0 +1,214 @@
+		appleNewsSetCSS( '.apple-news-settings-preview blockquote', 'blockquote_border_width', 'border-left-width', 'px', null );
+		appleNewsSetCSS( '.apple-news-settings-preview blockquote', 'blockquote_line_height', 'line-height', 'px', null );
+		appleNewsSetCSS( '.apple-news-settings-preview blockquote', 'blockquote_background_color', 'background-color', null, null );
+
+		// Monospaced
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_font', 'font-family', null, null );
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_size', 'font-size', 'px', null );
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_tracking', 'letter-spacing', 'px', $( '#monospaced_size' ).val() / 100 );
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_color', 'color', null, null );
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_line_height', 'line-height', 'px', null );
+		appleNewsSetCSS( '.apple-news-settings-preview pre', 'monospaced_line_height', 'margin-bottom', 'px', null );
+
+		// Component order
+		// This can either be defined as a sortable form element or a simple hidden element
+		var componentOrder;
+		if ( 0 === $( '#meta-component-order-sort' ).length && $( '#meta_component_order' ).length > 0 ) {
+			componentOrder = $( '#meta_component_order' ).val().split( ',' );
+		} else if ( $( '#meta-component-order-sort' ).length ) {
+			componentOrder = $( '#meta-component-order-sort' ).sortable( 'toArray' );
+			if ( '' !== componentKey ) {
+				$( '.apple-news-meta-component' ).removeClass( componentKey );
+				componentKey = '';
+			}
+		} else {
+			return;
+		}
+
+		$.each( componentOrder.reverse(), function( index, value ) {
+			// Remove the component
+			var $detached = $( '.apple-news-' + value ).detach();
+
+			// Build the component key.
+			// Used for targeting certain styles in the preview that differ on component order.
+			componentKey = value + '-' + componentKey;
+
+			// Add back at the beginning
+			$( '.apple-news-settings-preview' ).prepend( $detached );
+		} );
+
+		if ( '' !== componentKey ) {
+			componentKey = componentKey.substring( 0, componentKey.length - 1 );
+			$( '.apple-news-meta-component' ).addClass( componentKey );
+		}
+	}
+
+	function appleNewsSetCSS( displayElement, formElement, property, units, scale ) {
+		// Get the form value
+		var value = $( '#' + formElement ).val();
+
+		// If the value is 'none', make it empty
+		if ( 'none' === value ) {
+			value = '';
+		}
+
+		// Some values need to be scaled
+		if ( scale && value ) {
+			value = parseInt( value ) * scale;
+		}
+
+		// Add units if set and we got a value
+		if ( units && value ) {
+			value = value + units;
+		}
+
+		$( displayElement ).css( property, value );
+	}
+
+}( jQuery ) );
+
+function appleNewsSupportsMacFeatures() {
+	if ( 'MacIntel' === navigator.platform ) {
```

Comment:
Yes, this question has been asked before. Any other Mac platforms would not support the fonts required.